### DISCUSSION TITLE
Override Default value of "wt" Parameter in C#

### DISCUSSION THREAD

**Discussion by mahadeokashid** [SolrNet/SolrNet#628] [General]
Currently default value is "xml", which we want to change to "json". We tried using ExtraParams in QueryOptions as below 

ExtraParams = new Dictionary<string, string> { { "wt", "json" } }

however this does not work. it results in duplication of this "wt" parameter like 
select?q=*%3A*&fl=code%2Czoom&fq=coreName_s%3AMyCore&fq=code%3A(%22BUZ38SCFX%22)&wt=json&version=2.2&wt=xml

I want to see results in JSON format only, hence want to pass "wt=json". Help me to Fix this.

**Comment by mahadeokashid** [comment]
@mausch please guide in this case.

**Comment by mausch** [comment]
> Currently default value is "xml", which we want to change to "json".

Why? What are you trying to achieve?

**Reply by mahadeokashid** [reply]
I am trying to get response in json format from apache solr server using SolrNet library.

**Reply by mausch** [reply]
Why? SolrNet returns responses as .NET objects

**Comment by mausch** [comment]
People have asked for this in https://github.com/SolrNet/SolrNet/issues/572
though nobody has given any actual reason yet.
Closing as duplicate.
